Source/ASButtonNode.mm

```diff
@@ -52,7 +52,11 @@ - (ASTextNode *)titleNode
   ASLockScopeSelf();
   if (!_titleNode) {
     _titleNode = [[ASTextNode alloc] init];
-    // Intentionally not layer-backing the image node since tintColor may be applied
+    #if TARGET_OS_IOS
+          // tvOS needs access to the underlying view
+          // of the button node to add a touch handler.
+        [_titleNode setLayerBacked:YES];
```

Comment:
I know this is from the original impl, but the comment and the code don't add up. If the view is needed under tvOS, then this node should *not* be layer-backed under it (and can be layer-backed otherwise)?

Reply:
the comment might be misleading here, I think the intention was to gate this layer backing only for iOS and by doing nothing in tvOS it will automatically be view backed?

Reply:
I can make this clearer at least